### ISSUE TITLE
Foundation 6 reveal modal

### ISSUE THREAD

**Issue by ggornes** [HairdressingProject/HairdressingApp#1]
I'm trying to get a Reveal Modal component working as in [here]( https://get.foundation/sites/docs-v5/components/reveal.html)

On Dashboard page I included two click here links that are supposed to popup a modal when clicked.

**Comment by diego-cc** [comment]
The issue is that in order to use Foundation's modal component (from the base foundation-sites package), we would have to mix jQuery with React in this case. See [this post](https://stackoverflow.com/questions/36676780/react-component-foundation-modal-reveal-wont-open) for more info.

One solution that I found was to install a package that doesn't rely on jQuery, such as [react-foundation-modal](https://github.com/Shallyin83/react-foundation-modal). Check out the [master-d branch](https://github.com/HairdressingProject/HairdressingApp/tree/master-d), I have added a `Modal` component to `App/index.js`. It seems to work identically to Foundation's original modal component and it's also responsive.
